### ISSUE TITLE
Catalina OSX 10.15 support

### ISSUE THREAD

**Issue by cguentherTUChemnitz** [elliotweiser/ansible-osx-command-line-tools#53]
**What fails:**

```bash
TASK [elliotweiser.osx-command-line-tools : Check for Command Line Tools in Software Update list] ******************************************************************************************************************************************************************************
fatal: [vm-mac]: FAILED! => {"changed": false, "cmd": "set -o pipefail; softwareupdate -l | grep -B 1 -E 'Command Line Tools' | awk -F'*' '/^ +\\*/ {print $2}' | sed 's/^ *//' | grep -iE '[0-9|.]' | sort | tail -n1\n", "delta": "0:00:10.361220", "end": "2019-10-01 12:38:50.808701", "failed_when_result": true, "msg": "non-zero return code", "rc": 1, "start": "2019-10-01 12:38:40.447481", "stderr": "", "stderr_lines": [], "stdout": "", "stdout_lines": []}
```

**Why:**
`softwareupdate --list` seems to have changed output for mac OS catalina

Others do also suffer from this change:
https://github.com/microsoft/macos-cookbook/issues/202

(moved from https://github.com/geerlingguy/ansible-role-homebrew/issues/126)

**Comment by TheKas** [comment]
In Catalina this command fix it:
`softwareupdate -l | grep -B 1 -E 'Command Line Tools' | awk -F'*' '{print $2}'|sed 's/^ Label: //'|grep -iE '[0-9|.]'|sort |tail -n1`

**Comment by elliotweiser** [comment]
Hmm looks like that `Label: ` was added to the output. 

@TheKas -- Feel free to submit the PR so you can get credit :)

**Comment by TheKas** [comment]
the PR is on the way ;-)

**Comment by elliotweiser** [comment]
Looks like Ansible Galaxy hasn't added 10.15 to the platform list. Will bring that up with them separately. Not necessarily a blocker.

**Comment by elliotweiser** [comment]
Fixed in #54 and released in 2.3.0
